spec/models/miq_server_spec.rb

```diff
@@ -143,7 +142,7 @@
         config.config = {:ntp => server_ntp}
         config.save
 
-        MiqNtp.should_receive(:sync_settings).with(server_ntp)
+        @miq_server.should_receive(:apply_ntp_server_settings).with(server_ntp)
```

Comment:
Before, we were making expectations on us calling external code. Now, we are stubbing our own code.

Is it possible to change the expect on our external interface? e.g.:

```ruby
expect_any_instance_of(LinuxAdmin::Chrony).to receive(:clear)
expect_any_instance_of(LinuxAdmin::Chrony).to receive(:add_server).with(*server_ntp[:server])
```